.github/workflows/rust.yml

```diff
@@ -44,19 +47,24 @@ jobs:
         run: |
           sudo apt update
           sudo apt install -y protobuf-compiler libssl-dev
+          rustup update ${{ matrix.toolchain }} && rustup default ${{ matrix.toolchain }}
+          rustup component add rustfmt
       - name: Run cargo fmt
         run: cargo fmt --check
       # split everything us so we know what's slow.
       - name: Build
         run: |
           cargo build --all-features --tests
       - name: Start DynamoDB local for tests
+        if: ${{ matrix.toolchain == 'stable' }}
         run: |
           docker run -d -e AWS_ACCESS_KEY_ID=DUMMYKEY -e AWS_SECRET_ACCESS_KEY=DUMMYKEY -p 8000:8000 amazon/dynamodb-local
       - name: Run tests
+        if: ${{ matrix.toolchain == 'stable' }}
```

Comment:
should we still run some basic tests with nightly? Just to be safer with UBs, we do have a few `unsafe` in the codebase.

Reply:
Ok, add tests for nightly without extra features